conda_smithy/templates/run_osx_build.sh.tmpl

```diff
@@ -4,14 +4,17 @@ source .scripts/logging_utils.sh
 
 set -x
 
-startgroup "Installing a fresh version of Miniforge"
+{ set +x; startgroup "Installing a fresh version of Miniforge" } 2> /dev/null
```

Comment:
If you don't run this in a subshell, you alter the state of `x` option for all following code. Also misses a semicolon before `}`.
(But I guess you just missed changing this file since the others look correct :).)

----
But since you want to (and do) use subshells for which you throw away stderr, you can also simplify these to, e.g.,
```sh
( startgroup "Installing a fresh version of Miniforge" ) 2> /dev/null
```

Reply:
Oh, nice! I updated all the instances to the simplified version, thanks!